next/docs/iota-sdk/docs/how-tos/exchange-guide.mdx

```diff
@@ -0,0 +1,260 @@
+
+:::
+
+## Integration Guide
+
+This guide explains how to integrate the IOTA SDK in your exchange.
+
+Features of the IOTA SDK:
+
+- Secure seed management.
+- Account management with multiple accounts and multiple addresses.
+- Confirmation monitoring.
+- Deposit address monitoring.
+- Backup and restore functionality.
+
+### How Does it Work?
+
+The IOTA SDK is a stateful package with a standardized interface for developers to build applications involving value transactions. It offers abstractions to handle payments and can optionally interact with [Stronghold](https://wiki.iota.org/stronghold.rs/getting_started) for seed handling, seed storage, and state backup.
+
+:::note
+
+If you are not familiar with the IOTA SDK, you can find more information in the [documentation](../welcome.md).
+
+:::
+
+You can use the following examples as a guide to implementing _the multi-account approach_ using the `NodeJs` bindings:
+
+1. [Set up the the IOTA SDK](#1-set-up-the-iota-sdk).
+2. [Create an account for each user](#2-create-an-account-for-each-user).
+3. [Generate a user address to deposit funds](#3-generate-a-user-address-to-deposit-funds).
+4. [Check the user balance](#4-check-the-account-balance).
+5. [Listen to events](#5-listen-to-events).
+6. [Enable withdrawals](#6-enable-withdrawals).
+
+:::note
+
+If you are looking for other languages, please read the [IOTA SDK overview](../welcome/#your-application-in-the-iota-network).
+
+:::
+
+Since all IOTA SDK bindings are based on core principles provided by the IOTA SDK, the outlined approach is very similar regardless of the programming language you choose.
+
+<details>
+<summary> If you were already supporting Shimmer with wallet.rs </summary>
+
+The [IOTA SDK](https://github.com/iotaledger/iota-sdk) is a brand new project which is a combination and improvement of both [iota.rs](https://github.com/iotaledger/iota.rs) and [wallet.rs](https://github.com/iotaledger/wallet.rs).
+If you were already supporting Shimmer with wallet.rs, we encourage you to switch to the SDK as wallet.rs will be deprecated and not receive any update anymore.
+
+Most of the APIs are the same and the changes fairly minimal, here are some of the major changes:
+
+- The npm package changes from `@iota/wallet` to `@iota/sdk`
+- `AccountManager` becomes `Wallet`
+- `manager.generateMnemonic()` becomes `Utils.generateMnemonic()`
+- `generateAddress` becomes `generateEd25519Addresses`
+- `sendAmount` becomes `send`
+- The callbacks to `listen` now provide a properly typed `Event`
+
+</details>
+
+### 1. Set Up the IOTA SDK
+
+First, you should install the components that are needed to use the IOTA SDK and the binding of your choice; it may vary a bit from language to language. In the case of the [NodeJs binding](../getting-started/nodejs.mdx), it is straightforward since it is distributed via the `npm` package manager.
```

Comment:
```suggestion
First, you should install the components needed to use the IOTA SDK and the binding of your choice; it may vary a bit from language to language. In the case of the [NodeJs binding](../getting-started/nodejs.mdx), it is straightforward since it is distributed via the `npm` package manager.
```